dipy/segment/clustering.py

```diff
@@ -416,7 +417,7 @@ class QuickBundles(Clustering):
     metric : str or `Metric` object (optional)
         The distance metric to use when comparing two streamlines. By default,
         the Minimum average Direct-Flip (MDF) distance [Garyfallidis12]_ is
-        used and requires streamlines to have the same number of points.
+        used and streamlines are automatically resampled so they have 18 points.
```

Comment:
No, not really. I used 18 since that is what @Garyfallidis used in the example about streamlines segmentation. I don't know what you guys prefer: force the user to provide the number of points, or set one by default and mention it clearly in the documentation?